lib/PAUSE/dist.pm

```diff
@@ -9,6 +9,7 @@ use List::MoreUtils ();
 use PAUSE ();
 use Parse::CPAN::Meta;
 use PAUSE::mldistwatch::Constants;
+use JSON::XS ();
```

Comment:
PAUSE indexer is a standalone web service which is not expected to be run by others, and thus it doesn't need a wrapper at all. JSON::XS is the one that has been used by PAUSE for a long time, and it is already listed in PREREQ_PM in Makefile.PL. This usage is completely right.